src/linux/BTSerialPortBindingServer.cc

```diff
@@ -391,13 +372,11 @@ NAN_METHOD(BTSerialPortBindingServer::New) {
 
 
     BTSerialPortBindingServer *rfcomm = new BTSerialPortBindingServer();
+    uv_mutex_init(&rfcomm->mWriteQueueMutex);
+    ngx_queue_init(&rfcomm->mWriteQueue);
+    listen_baton_t * baton = rfcomm->mListenBaton;
     rfcomm->Wrap(info.This());
```

Comment:
where is info coming from?

Reply:
info is coming from the NAN_METHOD macro.  It's a parameter passed into all methods that are exposed to javascript.  It contains any arguments passed in from javascript, a pointer to this, and something to hold any return value(s).